<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="menu_admin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Opciones</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ver Residentes</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Agregar Residente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Actualizar Datos</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Estado Residente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Eliminar Residente</string>
    </property>
   </item>
  </widget>
  <widget class="QDialogButtonBox" name="boton_menuadmin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Garamond</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Men√∫ principal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
